packages/user/Supervisor/ui/src/utils.ts

```diff
@@ -56,6 +65,10 @@ export const parser = (): Promise<any> => {
     return modulePromise;
 };
 
+export let queryToken: string | undefined;
```

Comment:
I think technically this `queryToken` need not be exported if it's only used in these utils, right?